PythonScripts/ClipPDF.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# importing the required modules 
+import PyPDF2 
+import os
+from os import listdir
+from os.path import isfile, join
+
+#Clip PDFs to size 300kb
+def clipPDF(pdf): 
+    pdfFileObj = open(pdf, 'rb')
+    b = os.path.getsize(pdf)
+    pdfReader = PyPDF2.PdfFileReader(pdfFileObj)
+    pages = pdfReader.numPages 
+    if pages < 3:
+        return
+    end = 0
+    if b < 300000:
+        end = pages
+    else:
+        end = int((pages)*(300000/b))
```

Comment:
Are all pages of pdf supposed to be of equal size?

Reply:
No, but I wasn't able to find the size of each page using the library that I was using. Hence, I used an approximate to decide the number of pages to include.

Reply:
Done.